commercial/app/model/hosted/HostedAmp.scala

```diff
@@ -19,12 +19,24 @@ object HostedAmp {
       }
     }
 
+    def transformVideos(): Unit = {
+      def transformVideo(video: Element): Element = video.tagName("amp-video").attr("layout", "responsive").attr("width", "16").attr("height", "9")
+      val figures = doc.select("figure:has(gu-atom)")
+      for (figure <- figures.asScala;
+           video <- figure.select("video").asScala) {
+
+        figure.appendChild(transformVideo(video))
+        figure.select("gu-atom").remove()
```

Comment:
The `removeAtoms` function removes the whole `figure` tag (and the `amp-video` inside it). I looked at modifying that function but seemed cleaner to keep it a separate operation.